lib/foreman/renderer/scope/report.rb

```diff
@@ -44,6 +66,36 @@ def report_render_csv
           end
         end
 
+        def report_render_html
+          html = ""
+
+          html << "<html><head><title>#{@template_name}</title><style>#{html_style}</style></head><body><table><thead><tr>"
+          html << @report_headers.map { |header| "<th>#{ERB::Util.html_escape(header)}</th>" }.join('')
+          html << "</tr></thead><tbody>"
+
+          @report_data.each do |row|
+            html << "<tr>"
+            html << row.map { |cell| "<td>#{ERB::Util.html_escape(cell)}</td>" }.join('')
+            html << "</tr>"
+          end
+          html << "</tbody></table></body></html>"
+
+          html
+        end
+
+        def html_style
+          <<CSS
+th {
+    background-color: black;
+    color: white;
+}
+table,th,td {
+    border-collapse: collapse;
+    border: 1px solid black;
+}
```

Comment:
Couldn't we format this to match the block?
I know it's going to be few additional bytes and bit weird in raw report so it's no blocker, but I would prefer tidy codebase :thinking: What do you think?

Additionaly we could inline it and posibly use string concat:
```suggestion
th { background-color: black; color: white; }
table,th,td { border-collapse: collapse; border: 1px solid black; }
```

Reply:
I made it inline which looks better. We would still have to keep EOS on the beginning of the line, so I kept it starting on 0 column, it's not the first time we use this syntax